packages/components/src/breadcrumbs/index.stories.tsx

```diff
@@ -49,3 +58,22 @@ export const WithLongPath: Story = {
 		],
 	},
 };
+
+export const WithCustomItem: Story = {
+	args: {
+		...Default.args,
+		renderItemLink: ( { label, href, ...props } ) => {
+			const onClick = ( event: React.MouseEvent< HTMLAnchorElement > ) => {
+				props.onClick?.( event );
+				event.preventDefault();
+				alert( `Router navigation to ${ href }` );
```

Comment:
Can a `console.log()` be a better and less invasive alternative? Or maybe some inline notices?